packages/foam-vscode/src/test/suite.ts

```diff
@@ -38,6 +39,7 @@ export function run(): Promise<void> {
   //   throw err;
   // });
 
+  // eslint-disable-next-line no-async-promise-executor
```

Comment:
TIL https://eslint.org/docs/rules/no-async-promise-executor

I have actually tried to make the `run` function `async` and remove the `Promise` wrapper, and things ran well.. I wonder why this was done in the first place, but maybe instead of ignoring the rule we should remove the promise, wdyt?

Reply:
Same comment than before:

Yes, I'd like to change it, but I'm trying to avoid non-trivial refactors in this PR. That's why I've disabled some failing checks. I think it's an overall gain anyway: in the future we will be better advised not to do some things, while we still have a mark (we can search for eslint-disable) for previous violations of the rule. Maybe I can locally make this a warning instead of disabling it, I'm not sure if that's possible.

Reply:
It's only possible using a cumbersome syntax that will require to first turn the rule into a warning and then, in another line, revert that setting. I think it's better to keep the oneliner